src/dropdown/dropdown.js

```diff
@@ -170,9 +162,50 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
         return;
       }
       dropdownCtrl.dropdownMenu = element;
+
+      var dropdownMenu = element,
+          dropdown = element.parent(),
```

Comment:
This would potentially be a breaking change, if the dropdown element is not the parent element itself.